edx_rest_framework_extensions/auth/jwt/decoder.py

```diff
@@ -173,18 +173,73 @@ def _set_filters(token):
 
 
 def _verify_jwt_signature(token, jwt_issuer, decode_symmetric_token):
+    """
+    Verifies the JWT signature. Raises InvalidTokenError in the event of an error.
+
+    Arguments:
+        token (str): JWT to be decoded.
+        jwt_issuer (dict): A dict of JWT issuer related settings, containing the symmetric key.
+        decode_symmetric_token (bool): Whether to decode symmetric tokens or not. Pass False for asymmetric tokens only
+    """
+    # .. custom_attribute_name: jwt_auth_decode_symmetric_token
+    # .. custom_attribute_description: True if symmetric keys will also be used for checking
+    #   the JWT signature, and False if only asymmetric keys will be used.
+    set_custom_attribute('jwt_auth_decode_symmetric_token', decode_symmetric_token)
```

Comment:
nit: This reads a little misleadingly to me; I would expect it to mean that the token _was_ decoded using a symmetric key, not that the symmetric key is allowed to be used. (Non-blocking.)

Reply:
Updated to `jwt_auth_check_symmetric_key`.